src/libs/ValidationUtils.js

```diff
@@ -336,6 +337,25 @@ function doesFailCharacterLimitAfterTrim(maxLength, valuesToBeValidated) {
     return _.map(valuesToBeValidated, value => value && value.trim().length > maxLength);
 }
 
+/**
+ * Checks if input value includes comma or semicolon which are not accepted
+ *
+ * @param {String[]} valuesToBeValidated
+ * @returns {String[]}
+ */
+function findInvalidSymbols(valuesToBeValidated) {
+    return _.map(valuesToBeValidated, (value) => {
+        if (!value) {
+            return '';
+        }
+        let inValidSymbol = value.replace(/[, ]+/g, '') !== value ? Localize.translateLocal('personalDetails.error.comma') : '';
+        if (_.isEmpty(inValidSymbol)) {
+            inValidSymbol = value.replace(/[; ]+/g, '') !== value ? Localize.translateLocal('personalDetails.error.semicolon') : '';
+        }
+        return inValidSymbol;
```

Comment:
We will always get `invalidSymbol` here if we have spaces in the name @mountiny @sketchydroide.

Reply:
So obvious haha